README.md

```diff
@@ -21,11 +22,7 @@ Install-Package Elasticsearch.Net.Aws-v1
 // for NEST
 
 // if using an access key
-var httpConnection = new AwsHttpConnection("us-east-1", new StaticCredentialsProvider(new AwsCredentials
-{
-	AccessKey = "My AWS access key",
-	SecretKey = "My AWS secret key",
-}));
+var httpConnection = new AwsHttpConnection("us-east-1", new BasicAWSCredentials("My AWS access key", "My AWS secret key"));
```

Comment:
I updated the README, but not the csproj because I can see that you're experimenting with NEST 5. I didn't want to burn a major version number in case you decided that releasing that and this together was a good idea.